test/extended/util/framework.go

```diff
@@ -235,7 +236,7 @@ func WaitForADeployment(client kclient.ReplicationControllerInterface,
 			}
 		}
 	}
-	return fmt.Errorf("the deploy did not finish within 3 minutes")
+	return fmt.Errorf("the deploy did not finish within %v minutes", maxWaitingTime)
```

Comment:
He corrected it so the message prints the correct amount of time (15 minutes).  Making it a parameter is a nice addition.